djangocms_maps/templates/djangocms_maps/maps.html

```diff
@@ -16,8 +18,9 @@
         data-street_view_control="{{ object.street_view_control|lower }}"
         data-layers_control="{{ object.layers_control|lower }}"
         data-scale_bar="{{ object.scale_bar|lower }}"
-        data-lat="{% if object.lat %}{{ object.lat }}{% endif %}"
-        data-lng="{% if object.lng %}{{ object.lng }}{% endif %}"
+        {% if object.lat and object.lng %}
+            data-latlng="{{ object.lat }},{{ object.lng }}"
+        {% endif %}
```

Comment:
I'm afraid, this will break the implementation of the other map providers.

Also, please make sure the indenting of the resulting output is consistent and there are no superflous line breaks.